homeassistant/components/google_tasks/types.py

```diff
@@ -1,19 +1,17 @@
 """Types for the Google Tasks integration."""
 
 from dataclasses import dataclass
-from typing import Any
 
 from homeassistant.config_entries import ConfigEntry
 
-from .api import AsyncConfigEntryAuth
+from .coordinator import TaskUpdateCoordinator
 
 
 @dataclass
 class GoogleTasksData:
     """Class to hold Google Tasks data."""
 
-    api: AsyncConfigEntryAuth
-    task_lists: list[dict[str, Any]]
+    coordinators: dict[str, TaskUpdateCoordinator]
```

Comment:
If you're only storing one field in the dataclass, might as well just use that directly in the config entry

Reply:
Updated.